docs/otlp-source.md

```diff
@@ -7,15 +7,15 @@ The data needed to be converted to the formats the generic HTTP source supports
 Recently, we've added support for directly sending data using the OTLP protocol to Sumo Logic. This is achieved using the [OTLP
 source][otlp_source]. This document explains how to use this new source with the Helm Chart.
 
-**Note:** This Chart currently only supports OTLP Source for the following signal types: logs
+**Note:** This Chart currently only supports OTLP Source for the following signal types: logs, events
 
 ## Enabling the OTLP source
 
 Add the following to your configuration:
 
 ```yaml
 sumologic:
-  logs:
+  logs|events:
```

Comment:
This can confuse users. Please describe present separate snippets for logs and for events.